internal/consensus/pbts_test.go

```diff
@@ -521,6 +532,9 @@ func TestPBTSTooFarInTheFutureProposal(t *testing.T) {
 		height2ProposedBlockOffset:        100 * time.Millisecond,
 		height2ProposalTimeDeliveryOffset: 10 * time.Millisecond,
 		height4ProposedBlockOffset:        150 * time.Millisecond,
+		pbtsParams: types.PBTSParams{
+			PBTSEnableHeight: 1,
+		},
```

Comment:
Do we need another unit test here (or maybe adapt an existing one) to test activation? (i.e., PBTS enable height > 1) Or non-activation? (PBTS enable height stays at 0)

Reply:
We need a test, but that should be done in the base branch, which adds the parameter.